src/db/tables/subscribers.ts

```diff
@@ -390,41 +406,54 @@ async function getPotentialSubscribersWaitingForFirstDataBrokerRemovalFixedEmail
  */
 // Not covered by tests; mostly side-effects. See test-coverage.md#mock-heavy
 /* c8 ignore start */
-async function getSubscribersWaitingForMonthlyEmail (options = {}) {
+async function getSubscribersWaitingForMonthlyEmail(
+  options: Partial<{ plusOnly: boolean; limit: number }> = {},
+): Promise<SubscriberRow[]> {
   // I'm explicitly referencing the type here, so that these lines of code will
   // show up as errors when we remove it from the flag list:
-  /** @type {import("./featureFlags.js").FeatureFlagName} */
   const featureFlagName = "MonthlyActivityEmail";
   // Interactions with the `feature_flags` table would generally go in the
   // `src/db/tables/featureFlags` module. However, since that module is already
   // written in TypeScript, it can't be loaded in pre-TypeScript cron jobs,
   // which currently still import from the subscribers module. Hence, we've
   // inlined this until https://mozilla-hub.atlassian.net/browse/MNTOR-3077 is fixed.
-  const flag = (await knex("feature_flags")
-      .first()
-      .where("name", featureFlagName)
-      // The `.andWhereNull` alias doesn't seem to exist:
-      // https://github.com/knex/knex/issues/1881#issuecomment-275433906
-      .whereNull("deleted_at"));
+  const flag = await knex("feature_flags")
+    .first()
+    .where("name", featureFlagName)
+    // The `.andWhereNull` alias doesn't seem to exist:
+    // https://github.com/knex/knex/issues/1881#issuecomment-275433906
+    .whereNull("deleted_at");
 
   if (!flag?.is_enabled) {
     return [];
   }
 
-  let query = knex('subscribers')
+  let query = knex("subscribers")
     .select()
     // Only send to users who haven't opted out of the monthly activity email...
-    .where((builder) => builder.whereNull("monthly_monitor_report").orWhere("monthly_monitor_report", true))
+    // eslint-disable-next-line @typescript-eslint/no-misused-promises
```

Comment:
`eslint-disable` comments are surprising, so we can help future readers by explaining it. Same for the one on line 441.

```suggestion
    // (It looks like Knex's `.where` type definition doesn't accept Promise-returning
    // functions, even though the code does; hence the `eslint-disable`)
    // eslint-disable-next-line @typescript-eslint/no-misused-promises
```